### ISSUE TITLE
bruter.py and display.py syntax errors

### ISSUE THREAD

**Issue by 1hemm** [Pure-L0G1C/Instagram#145]
Traceback (most recent call last):
  File "instagram.py", line 7, in <module>
    from lib.bruter import Bruter
  File "/mnt/c/Users/harsh shah/desktop/Instagram/lib/bruter.py", line 7, in <module>
    from lib.display import Display
  File "/mnt/c/Users/harsh shah/desktop/Instagram/lib/display.py", line 74
    f'[-] Wordlist: {self.passlist}\n[-] Username: {self.username}\n[-] Password: {password}')
                                                                                            ^
SyntaxError: invalid syntax

**Comment by Pure-L0G1C** [comment]
What version of Python are you using? You should be using at least Python 3.6 at the moment

**Comment by anton-serozhechkin** [comment]
Im using python version 3.8

вс, 26 апр. 2020 г., 23:59 Mohamed <notifications@github.com>:

**Comment by 1hemm** [comment]
i am using Python v3.8

> What version of Python are you using? You should be using at least Python 3.6 at the moment

**Comment by Pure-L0G1C** [comment]
f-strings shouldn't be an issue in Python3.8

**Comment by 1hemm** [comment]
can i know step by step usage of script?
i think i am doing any mistake!

**Comment by 1hemm** [comment]
Let me try again..
if problem exists, i will let you know, just a moment
and one more thing, i am using windows terminal for this.

**Comment by 1hemm** [comment]
whole process, what i did! 
this is the problem i am facing!



TheMM@HarshHP:/mnt/c/Users/harsh shah$ sudo su
[sudo] password for TheMM:
root@HarshHP:/mnt/c/Users/harsh shah# cd desktop
root@HarshHP:/mnt/c/Users/harsh shah/desktop# cd instagram
root@HarshHP:/mnt/c/Users/harsh shah/desktop/instagram# ls
database  instagram.py  lib  LICENSE  password1.txt  README.md  requirements.txt  test_proxies.py
root@HarshHP:/mnt/c/Users/harsh shah/desktop/instagram# python3 instagram.py hrs.hah password1.txt -m 0
Traceback (most recent call last):
  File "instagram.py", line 7, in <module>
    from lib.bruter import Bruter
  File "/mnt/c/Users/harsh shah/desktop/instagram/lib/bruter.py", line 7, in <module>
    from lib.display import Display
  File "/mnt/c/Users/harsh shah/desktop/instagram/lib/display.py", line 73
    print(f'[-] Wordlist: {self.passlist}\n[-] Username: {self.username}\n[-] Password: {password}')
                                                                                                  ^
SyntaxError: invalid syntax
root@HarshHP:/mnt/c/Users/harsh shah/desktop/instagram#

**Comment by 1hemm** [comment]
how i can replace python 3.5 to python 3.8 in ubuntu as default?

**Comment by n45743673578864677** [comment]
how can I hack someone inestagram?
